templates_ark/example_neighborhood_analysis_script.ipynb

```diff
@@ -1,18 +1,32 @@
 {
```

Comment:
<p>Grande text size again</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Same as above, it looks fine on github (https://github.com/angelolab/ark-analysis/blob/kmeans_neighborhood_nb_update/templates_ark/example_neighborhood_analysis_script.ipynb) and looks fine in my jupyter.